tests/integration/testdata/sync/code/before/template-python.yaml

```diff
@@ -16,6 +16,24 @@ Resources:
         - Ref: HelloWorldLayer
```

Comment:
This file name is no longer valid anymore (`template-python.yaml`). Let's move this function into its own template.